build/commands/lib/config.js

```diff
@@ -565,6 +572,13 @@ Config.prototype.buildArgs = function () {
     args.symbol_level = 1
   }
 
+  if (this.isCoverage()) {
+    args.symbol_level = 1
+    args.use_clang_coverage = true
+    args.use_clang_profiling_inside_sandbox = true
```

Comment:
this appears to already be set by default when `use_clang_coverage == true`

Reply:
my previous tests didn't suggest that. The no profiles were generated and error messages were thrown. But maybe it changed in the last week?

Reply:
were you cross compiling maybe?

Reply:
either way it seems like we should be setting `use_clang_profiling` instead of `use_clang_profiling_inside_sandbox` if it's needed?

Reply:
> were you cross compiling maybe?

arent we always cross compiling given that all our RBE nodes are exclusively linux-x64?

Reply:
cc @atuchin-m